src/main/java/org/spongepowered/api/data/meta/Enchantment.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.data.meta;
+
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.data.DataSerializable;
+import org.spongepowered.api.data.persistence.DataBuilder;
+import org.spongepowered.api.item.EnchantmentType;
+import org.spongepowered.api.item.inventory.ItemStack;
+
+/**
+ * Represents an {@link EnchantmentType} on an {@link ItemStack} that is paired
+ * with a level.
+ *
+ * <p>The contract of enchantments is that their level will
+ * never be below 1.</p>
```

Comment:
````
<p>The contract of enchantments is that their level will
be in the range of {@code getType().getMinimumLevel()}
and {@code getType().getMinimumLevel()}.</p>
````

Reply:
Uhh yes, it can be any natural number as long as the number can be stored in data.
Does this hurt items with empty enchantment tags? Those tags may give item stacks an effect while not having a valid enchantment.

Reply:
So we only check when building if it is above 0 or below Short.MAX_VALUE. There are issues if the level is below 0 if I recall correctly but in reality it can be 0(I think?). I just changed it to 1 since I didn't see a reason why an enchantment would ever need to be 0 and Minecraft doesn't really expect it with all of theirs having a minimum level of 0. Do you think I should switch back to 0 or greater?